src/Utils/axiosWithAuth.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const axiosWithAuth = () => {
```

Comment:
Another way to do this since this is the only thing we're exporting we can do 
`export const axiosWithAuth`

but this does mean you can't rename it when importing